modules/nodes-scheduler-taints-tolerations-adding-machineset.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-scheduler-taints-tolerations.adoc
+// * post_installation_configuration/node-tasks.adoc
+
+[id="nodes-scheduler-taints-tolerations-adding-machineset_{context}"]
+= Adding taints and tolerations using a MachineSet
+
+You can add taints to nodes using a Machineset. All nodes associated with the `MachineSet` are updated with the taint. Pods and tolerations respond the same as a taint added directly to the nodes.
```

Comment:
_Pods and tolerations respond the same as a taint added directly to the nodes._

?

Reply:
_Tolerations respond to taints added by a machine set in the same manner as taints added directly to the nodes._